test/charts/topology/topology.spec.js

```diff
@@ -164,7 +166,8 @@ describe('Component: pfTopology', function() {
 
     var icons = $scope.data.icons;
     $scope.nodes = {};
-    for(var kind in nodeKinds) {
+
+    Object.keys(nodeKinds).forEach(function (kind) {
```

Comment:
Altered this loop since it was linting towards something like.... Can alter it back towards this if we vote this direction

```js
for(var kind in nodeKinds) {
  if (nodeKinds.hasOwnProperty(kind)) {
...
```

Reply:
I don't have any particular preference. I'm fine with either.